README.md

```diff
@@ -539,6 +539,136 @@ val jsonSchema: ASchema = TapirSchemaToJsonSchema(
 )
 ```
 
+#### Generating JSON Schema from case class
+
+We can also generate JSON Schema directly from case class, without defining the schema manually:
```

Comment:
I would expand this a bit:
* some more detail on how this differs from the example immediately above
* first mention the methods that the section focuses on (as I understand, this is about `SchematizedFunctionTool`?), describe how this works, and only later proceed to the example. The example is long and it might be hard understanding what's the main point